packages/cli/src/utils/require-utils.ts

```diff
@@ -13,7 +13,37 @@ export const requireJsCode = <T>(code: string): T => {
 
   const m = new Module(fileid)
   m.filename = filename
-  // @ts-ignore
-  m._compile(code, filename)
-  return m.exports
+
+  try {
+    // @ts-ignore
+    m._compile(code, filename)
+    return m.exports
+  } catch (compileError: unknown) {
+    throw _injectStackTrace(compileError as Error, code, filename)
+  }
+}
+
+const STACK_TRACE_SURROUNDING_LINES = 3
+
+const _injectStackTrace = (compileError: Error, code: string, filename: string) => {
+  if (!compileError.stack || !compileError.stack.includes(`${filename}:`)) {
+    return compileError
+  }
+
+  // Extract line and column from the stack trace:
+  const [, locationInfo] = compileError.stack.split(`${filename}:`, 2)
+  const [lineNoStr, _rest] = locationInfo!.split(':', 2)
```

Comment:
nodejs stack traces are always formatted the same way, so the following uses of `!` are actually *safe*